src/router/index.js

```diff
@@ -61,13 +67,6 @@ const routes = [
     component: () => import("@/pages/Error"),
     props: { type: "404" },
   },
-  {
-    // Refer to: https://stackoverflow.com/a/64186073/7870587
-    path: "/:pathMatch(.*)*",
-    redirect: {
-      name: "404",
-    },
-  },
```

Comment:
why remove this? We kept this to redirect the user to 404 if they are visiting any URL not matching the routes above.

Reply:
sorry was experimenting. Putting it back.